chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CacheInitializorImpl.java

```diff
@@ -0,0 +1,49 @@
+package gov.healthit.chpl.domain;
+
+import java.io.IOException;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.test.context.ContextConfiguration;
+
+import gov.healthit.chpl.auth.permission.GrantedPermission;
+import gov.healthit.chpl.auth.user.JWTAuthenticatedUser;
+import gov.healthit.chpl.dao.EntityRetrievalException;
+import gov.healthit.chpl.web.controller.CertificationIdController;
+import gov.healthit.chpl.web.controller.SearchViewController;
+
+@ContextConfiguration(classes = { gov.healthit.chpl.CHPLConfig.class })
+@Component
+public class CacheInitializorImpl {
+	private static JWTAuthenticatedUser adminUser;
+	private static final Logger logger = LogManager.getLogger(CacheInitializorImpl.class);
+	
+	@Autowired private CertificationIdController certificationIdController;
+	@Autowired private SearchViewController searchViewController;
+	
+	@Async
+	public void initializeSearchOptions() throws IOException, EntityRetrievalException {
```

Comment:
Can you cache the `getPopulateSearchData(true)` call as well? I think that's the one that's used in the admin section, and I'd love it if that were faster :)